server/package.json

```diff
@@ -21,7 +21,7 @@
     "migrate": "prisma migrate deploy",
     "migrate:reset": "dotenv -e ./dotenv/.env -- prisma migrate reset",
     "migrate:dev": "dotenv -e ./dotenv/.env -- prisma migrate dev",
-    "migrate:deploy": "dotenv -e ./dotenv/prod.env -- prisma migrate deploy",
+    "migrate:deploy": "dotenv -e ./dotenv/.env -- prisma migrate deploy",
```

Comment:
@hyochan What do you think about moving `dotenv/.env` to `./.env` and let prisma cli automatically load the env variables? This way we can remove `migrate:something` scripts from package.json and just do:
```bash
yarn prisma migrate deploy
yarn prisma migrate dev
# etc...
```
Reference: https://www.prisma.io/docs/concepts/more/environment-variables

Reply:
Actually, we moved to `dotenv` because we didn't want to organize two `env` files separately. We still need other env variables for the application.